scss/barrier.scss

```diff
@@ -60,15 +62,68 @@ $barrier-padding: 20px;
 	text-align: center;
 }
 
-.amp-access-barrier__offer-price {
-	@include oTypographySans(m);
+.amp-access-barrier__offer-secondary-price {
+	color: oColorsGetPaletteColor('grey-tint3');
+	font-size: 14px;
+	line-height: 16px;
+}
+
+.amp-access-barrier__offer-promo-separator {
+	position: relative;
+    text-align: center;
+	margin: 0 0 8px;
+
+	&:before, &:after {
+		background-color: oColorsGetPaletteColor('black');
+		content: '';
+		display: block;
+		height: 1px;
+		opacity: 0.25;
+		position: absolute;
```

Comment:
would it be possible to put positioning attributes and attributes that have the biggest change to the size and position of the element at the top of the scope?